controllers/index.js

```diff
@@ -10,10 +10,47 @@ router.get('/', async (req, res, next) => {
     // Fetch templates
     const {templates} = await model.list(0, null);
     const filterOptions = {
-      sort: req.query.sort || 'views',
-      templateTypes: (req.query.templateTypes || 'all').split(','),
-      categories: (req.query.categories || 'all').split(',')
+      sort: (req.query.sort) ? req.query.sort.split(',') : ['views'],
+      templateTypes: (req.query.templateTypes) ? req.query.templateTypes.split(',') : ['all'],
+      categories: (req.query.categories) ? req.query.categories.split(',') : ['all']
     };
```

Comment:
No need for parentheses around `req.query.*`.

Reply:
removed